static/themes/enterprise-runtime.css

```
@@ -0,0 +1,210 @@
+  --tracking-tight: -0.4px;
+
+  --text-xs: 0.825rem;
+  --text-sm: 0.925rem;
+  --text-base: 1.125rem;
+
+  /* radius variables */
+  --radius-xxs: 1px;
+  --radius-xs: 1px;
+  --radius-sm: 1px;
+  --radius: 4px;
+  --radius: 4px;
```

Biome (2.1.2)

[error] Duplicate custom properties can lead to unexpected behavior and may override previous declarations unintentionally.

--radius is already defined here.

Remove or rename the duplicate custom property to ensure consistent styling.

(lint/suspicious/noDuplicateCustomProperties)